pipit/trace.py

```diff
@@ -29,3 +31,77 @@ def from_hpctoolkit(dirname):
         from .readers.hpctoolkit_reader import HPCToolkitReader
 
         return HPCToolkitReader(dirname).read()
+
+    def comm_matrix(self, comm_type="bytes"):
```

Comment:
we should change comm_type to output and the values we should allow are count or size